packages/runtime/container-runtime/src/summary/runningSummarizer.ts

```diff
@@ -56,7 +56,7 @@ const defaultNumberSummarizationAttempts = 2; // only up to 2 attempts
  */
 export class RunningSummarizer implements IDisposable {
 	public static async start(
-		logger: ITelemetryLoggerExt,
```

Comment:
Feels like within the runtime we should be fine to pass around `ITelemetryLoggerExt`, why the change?

Reply:
I found the existing convo, got it.